src/common/collection.c

```diff
@@ -2255,6 +2255,7 @@ void dt_collection_hint_message(const dt_collection_t *collection)
   }
 
   g_idle_add(dt_collection_hint_message_internal, message);
+  g_free(message);
```

Comment:
pretty sure this is wrong, message is a pointer passed to dt_collection_hint_message_internal callback. If you free it there for sure it will crash at some point.

Reply:
Yes, there are a places like this where I'm not sure either as I'm not familiar w/ what will GLib/GTK do...